### PR TITLE
addedNewPowerUpIcons

### PR DESCRIPTION
**Pull request by ferinator**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,5 +55,10 @@
     "lerna": "^3.20.2",
     "prettier": "1.19.1",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
```
Comment (StuckiSimon) [inline package.json]: Quick code scan note: This probably shouldn't be here 😅

File: game-frontend/src/components/canvas-objects/GameObjects.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   Consumable as ConsumableT,
   GameObjectTypes,
 } from '@darwin/types';
 import Unit from './Unit';
 import Food from './Food';
-import PowerUp from './Powerup';
+import { Teleport, Endurance, HealthRegen } from './Powerup';
 import scalePosition from '../../helper/scalePosition';
 import Wall from './Wall';
+import { GameObject } from '../../../../darwin-types/dist/game-objects/GameObject';
```
Comment (mailaenderli) [inline game-frontend/src/components/canvas-objects/GameObjects.tsx]: Shouldn't you use here ``import { GameObjects } from '@darwin/types';``?

File: game-frontend/package.json [hunk 3]
```diff
@@ -49,7 +49,8 @@
     "styled-components": "^5.0.1",
     "typescript": "~3.7.2",
     "use-persisted-state": "^0.3.0",
-    "uuid": "^7.0.3"
+    "uuid": "^7.0.3",
+    "react-update": "^0.4.4"
```
Comment (sir-marc) [inline game-frontend/package.json]: What's that for?

File: game-frontend/src/components/canvas-objects/Unit.tsx [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import React, { FC } from 'react';
```
Comment (sir-marc) [inline game-frontend/src/components/canvas-objects/Unit.tsx]: I would not change the unit visualization.
I think adding icons for the powerups should be enough.

File: game-frontend/src/components/canvas-objects/Powerup.tsx [hunk 5]
```diff
@@ -1,46 +1,55 @@
...
 };
 
-const HALF_FIELD_SIZE = FIELD_SIZE / 2;
-
-const squareScale = 0.4;
-const squareSize = FIELD_SIZE * squareScale;
-const squareMargin = (FIELD_SIZE - squareSize) / 2;
-
-const radius = HALF_FIELD_SIZE * 0.3;
+const HealthRegen: FC<Props> = ({ position }) => {
```
Comment (sir-marc) [inline game-frontend/src/components/canvas-objects/Powerup.tsx]: What about adding the icon inside the colored circle? So it is better aligned with our overall art-style.
Also i would suggest to use uni-colored icons.

Comment (ferinator) [reply game-frontend/src/components/canvas-objects/Powerup.tsx]: i didn't manage to do this, maybe if you are familiar you can add the necessary change and i will take a look and learn :)

**Comment by michaelschufi** [comment]
@ferinator @sir-marc 
I'm getting an error when I try to test it:
`Error: The SVG image must have width and height defined (in pixels), canvas API needs them.`

**Comment by ferinator** [comment]
> height

Difficult to say what the problem is, since it works on my side..

File: game-frontend/package-lock.json [hunk 6]
```diff
@@ -11686,6 +11702,15 @@
         "tslib": "^1.0.0"
       }
     },
+    "react-update": {
```
Comment (StuckiSimon) [inline game-frontend/package-lock.json]: @sir-marc When reverting package.json always also revert the lock file. This lock file contains entries which shouldn't be there. (such as this one)

**Comment by sir-marc** [comment]
image

**Review by michaelschufi — APPROVED** [review]

**Pull request merged**